packages/atomic/src/components/search/atomic-search-box/atomic-search-box.new.stories.tsx

```diff
@@ -43,3 +47,385 @@ export const RichSearchBox: Story = {
+                  </atomic-result-section-bottom-metadata>
+                  <atomic-table-element label="Description">
+                    <atomic-result-section-title
+                      ><atomic-result-link></atomic-result-link
+                    ></atomic-result-section-title>
+                    <atomic-result-section-title-metadata>
+                      <atomic-field-condition
+                        class="field"
+                        if-defined="snrating"
+                      >
+                        <atomic-result-rating
+                          field="snrating"
+                        ></atomic-result-rating>
+                      </atomic-field-condition>
+                      <atomic-field-condition
+                        class="field"
+                        if-not-defined="snrating"
+                      >
+                        <atomic-result-printable-uri
+                          max-number-of-parts="3"
+                        ></atomic-result-printable-uri>
+                      </atomic-field-condition>
+                    </atomic-result-section-title-metadata>
+                    <atomic-result-section-excerpt>
+                      <atomic-result-text field="excerpt"></atomic-result-text>
+                    </atomic-result-section-excerpt>
+                  </atomic-table-element>
+                  <atomic-table-element label="author">
+                    <atomic-result-multi-value-text
+                      field="author"
+                    ></atomic-result-multi-value-text>
+                  </atomic-table-element>
+                  <atomic-table-element label="source">
+                    <atomic-result-text field="source"></atomic-result-text>
+                  </atomic-table-element>
+                  <atomic-table-element label="language">
+                    <atomic-result-multi-value-text
+                      field="language"
+                    ></atomic-result-multi-value-text>
+                  </atomic-table-element>
+                  <atomic-table-element label="file-type">
+                    <atomic-result-text field="filetype"></atomic-result-text>
+                  </atomic-table-element>
+                </template>
+              </atomic-result-template>
+            </atomic-result-list>
+            <atomic-query-error></atomic-query-error>
+            <atomic-no-results></atomic-no-results>
+          </atomic-layout-section>
+          <atomic-layout-section section="pagination">
+            <atomic-load-more-results></atomic-load-more-results>
+            <!-- Alternative pagination
+              <atomic-pager></atomic-pager>
+              <atomic-results-per-page></atomic-results-per-page>
+              -->
+          </atomic-layout-section>
+        </atomic-layout-section>
+      </atomic-search-layout>`,
+  ],
+  play: async (context) => {
+    await play(context);
+    await playExecuteFirstSearch(context);
```

Comment:
Why execute first search twice ? It gets executed in `play()` as well.

Reply:
It does not if `wrapInSearchInterface` second param is true ;) 
https://github.com/coveo/ui-kit/blob/4810401f3d18f5ecdcee0cb12a98102473040138/packages/atomic/storybookUtils/search/search-interface-wrapper.tsx#L10-L13